build/DotNetSdkVersions.txt

```diff
@@ -0,0 +1,5 @@
+# Each line represents arguments for the .NET SDK installer script (https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
+-Channel 7.0
+-Channel 6.0 -Runtime dotnet
```

Comment:
What's the purpose of the `-runtime dotnet` on the line, but not others?

We've had contributors using IBM's z360 mainframe contribute to NuGet, but it uses the mono runtime (not Mono Project, that implements .NET Framework, but the mono runtime of the .NET Core App runtime). By specifying `-runtime dotnet`, is it going to cause problems for customers on platforms that the dotnet runtime doesn't have binaries for?

Reply:
I found that some of our tests need the .NET 6 runtime but don't need the full SDK.  I've experimented a little and hope to trim this down as soon as all of our tests target .NET 7 or .NET 8.